build.sbt

```diff
@@ -14,26 +15,24 @@ lazy val scalaModels = project.in(file("."))
       "com.thesamet.scalapb" %% "scalapb-runtime" % scalapb.compiler.Version.scalapbVersion % "protobuf"
     ),
 
+    Compile / scalacOptions ++= Seq("-release:11"),
+
     Compile / PB.targets := Seq(
       scalapb.gen() -> (Compile / sourceManaged).value / "scalapb"
     ),
 
     Compile / PB.protoSources := Seq(baseDirectory.value / "./proto"),
     releaseCrossBuild := true,
-    releaseProcess := {
-      val process = Seq[ReleaseStep](
-        checkSnapshotDependencies,
-        inquireVersions,
-        runClean,
-        runTest,
-        setReleaseVersion,
-        releaseStepCommandAndRemaining("+publishSigned"),
-      )
-
-      if (!isSnapshot.value) {
-        process ++ Seq[ReleaseStep](releaseStepCommand("sonatypeBundleRelease"))
-      } else {
-        process
-      }
-    }
+    releaseVersion := fromAggregatedAssessedCompatibilityWithLatestRelease().value,
+    releaseProcess := Seq[ReleaseStep](
+      checkSnapshotDependencies,
+      inquireVersions,
+      runClean,
+      runTest,
+      setReleaseVersion,
+      commitReleaseVersion,
+      tagRelease,
+      setNextVersion,
+      commitNextVersion
```

Comment:
TODO: I think we'll need to update branch protection settings to allow the next version to be committed (to `main`)?

Reply:
Yes, I would say you can disable branch protection for now- see https://github.com/guardian/gha-scala-library-release-workflow/issues/5 & https://github.com/guardian/gha-scala-library-release-workflow/issues/5#issuecomment-1853802212 - DevX are very supportive about adopting `gha-scala-library-release-workflow`, and are happy for branch protection to be disabled for the time being while this is rolled out.